src/main/java/duke/Task.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+/**
+ * Represents a task in the list.
+ */
+public class Task {
+    String name;
+    boolean done;
```

Comment:
Would it be appropriate to perhaps make these variables private, if you will be providing setters anyways?